README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
   },
 );
 ``` 
+<br>
+
+### Apple Pay Component with Sessions flow
+1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
+- `sessionData`: the payment session data.
+- `id`: a unique identifier for the session data.
+- The request body.
+
+Pass these values to your app.
+
+2. In your Flutter app, create a `ApplePayComponentConfiguration` instance. It requires a `ApplePayConfiguration` which contains your merchant id and merchant name.
+
+```dart
+final ApplePayConfiguration applePayConfiguration = ApplePayConfiguration(
+  merchantId: MERCHANT_ID,
+  merchantName: MERCHANT_NAME,
+);
+
+final ApplePayComponentConfiguration applePayComponentConfiguration = ApplePayComponentConfiguration(
+  // Change the environment to live when you are ready to accept real payments.
+  environment: Environment.test
+  clientKey: CLIENT_KEY,
+  countryCode: COUNTRY_CODE,
+  amount: AMOUNT,
+  applePayConfiguration: applePayComponentConfiguration,
+);
+```
+3. Create a `SessionCheckout` by calling the `AdyenCheckout.session.create()` while passing the required properties:
+```dart
+final sessionCheckout = await AdyenCheckout.session.create(
+  sessionId: sessionResponse.id,
+  sessionData: sessionResponse.sessionData,
+  configuration: applePayComponentConfiguration,
+);
+```
+4. Get the Apple Pay payment method from the `sessionCheckout` object.
+5. Use the Apple Pay component widget:
+```dart
+AdyenApplePayComponent(
+  configuration: applePayComponentConfiguration,
+  paymentMethod: applePayPaymentMethod,
+  checkout: sessionCheckout,
+  style: ApplePayButtonStyle(width: 200, height: 48),
+  onPaymentResult: (paymentResult) {
+    //Handle the payment result
+  },
+),
+``` 
+<br>
+
+### Apple Pay Component with Advanced flow
+
+1. From your backend, make a [`/paymentMethods`](https://docs.adyen.com/api-explorer/Checkout/71/post/paymentMethods) request.
+2. Get the Apple Pay payment method by filtering the payment methods list.
+3. Create the `ApplePayComponentConfiguration`. It requires the `ApplePayConfiguration` which contains your merchant id and merchant name. You can also provide optional properties for an enhanced Apple Pay use case.
+
+```dart
+final ApplePayConfiguration applePayConfiguration = ApplePayConfiguration(
+  merchantId: MERCHANT_ID,
+  merchantName: MERCHANT_NAME,
+);
+
+final ApplePayComponentConfiguration applePayComponentConfiguration = ApplePayComponentConfiguration(
+  // Change the environment to live when you are ready to accept real payments.
+  environment: Environment.test
+  clientKey: CLIENT_KEY,
+  countryCode: COUNTRY_CODE,
+  amount: AMOUNT,
+  applePayConfiguration: applePayComponentConfiguration,
+);
+```
+4. Create an `AdvancedCheckoutPreview` providing two callbacks:
```

Comment:
same suggestion as the one I made for the cards version (to say what they have to do explicitly instead of "needs to be forwarded", except the extra description.